### PR TITLE
[DOCS] Adds default inference endpoints information

### PR DESCRIPTION
**Pull request by kosabogi**
### Overview

This PR adds the new default inference endpoints information to the [Inference APIs](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) page.

### Preview

[Inference APIs - Default Inference endpoints](https://elasticsearch_bk_118463.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/inference-apis.html)

### CODE REVIEW

File: docs/reference/inference/inference-apis.asciidoc [hunk 1]
```diff
@@ -48,21 +48,21 @@ When adaptive allocations are enabled:
...
-//[discrete]
-//[[default-enpoints]]
-//=== Default {infer} endpoints
+[discrete]
+[[default-enpoints]]
+=== Default {infer} endpoints
 
-//Your {es} deployment contains some preconfigured {infer} endpoints that makes it easier for you to use them when defining `semantic_text` fields or {infer} processors.
-//The following list contains the default {infer} endpoints listed by `inference_id`:
+Your {es} deployment contains some preconfigured {infer} endpoints that makes it easier for you to use them when defining `semantic_text` fields or {infer} processors.
```
Comment (leemthompo) [inline docs/reference/inference/inference-apis.asciidoc:55]:
```suggestion
Your {es} deployment contains preconfigured {infer} endpoints which makes them easier to use when defining `semantic_text` fields or using {infer} processors.
```

Comment (leemthompo) [reply docs/reference/inference/inference-apis.asciidoc:55]: minor rewording for clarity

**Review by leemthompo — APPROVED** [review]
LGTM

**Comment by szabosteve** [comment]
The PR check fails because [this PR](https://github.com/elastic/elasticsearch/pull/118463) hasn't been backported to 8.x. When the backport is done, the CI job can be restarted on this PR.

**Comment by kosabogi** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/119572)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**